index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+
+  <meta charset="UTF-8">
+  <title>Portfolio - Valentino Valkaj</title>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@300&family=Poppins&family=Roboto&display=swap" rel="stylesheet"/>
+  <link rel="stylesheet" type="text/css" href="style.css" />
+
+</head>
+
+<!--header-->
+<body>
+    <header id="header" class="header-scroll">
+        <nav class="nav">
+            <a href="index.html" class="my-logo">My Logo</a>
+            <span class="hamburger"></span>
+            <img src="./assets/icons8-hamburger-menu-64.png" class="navbar-hamb">
+
+        </nav>
+    </header>    
+
+        <!--main-->
+        <main id="main">
+            <section class="home-page">
+                <h1 class="h1">Hey There.<br />I'm Valentino.</h1>
+
+                <div class="description">
+                    <p class="h2-description">I'm a Software Developer</p>
+
+                    <p class="p-description">
+                        I can help you build a product, feature or website Look through some
+                        of my work and experience! If you like what you see and have a
+                        project you need coded, don't hesitate to contact me.
+                    </p>
+                </div>
+            </section>
```

Comment:
- [ ] As requested by the previous reviewer [here](https://github.com/valentino2105/Portfolio-setup-and-mobile-first/pull/1#discussion_r1214065945) kindly consider  adding a background image on this section to make your design an exact duplicate of the provided one check the images below to see the difference easily 
- your design 👇 
<img width="384" alt="Screen Shot 2023-06-02 at 20 01 00" src="https://github.com/valentino2105/Portfolio-setup-and-mobile-first/assets/60146030/919463f2-191a-4399-8c3d-56b88608392e">
 - Figma design 👇 
 
<img width="375" alt="Screen Shot 2023-06-02 at 20 02 08" src="https://github.com/valentino2105/Portfolio-setup-and-mobile-first/assets/60146030/9b88719f-6b21-4c74-aa9b-58281fd5c6a7">